app/models/creative.rb

```diff
@@ -99,4 +102,8 @@ def assign_wide_image(blob_id)
   def touch_campaigns
     campaigns.map(&:touch)
   end
+
+  def set_organization
+    self.organization = user.organization
+  end
```

Comment:
The common behavior for the `has_many` relations of `Organization` should perhaps get rolled into a concern. Some thing like:

```ruby
include Organizable
```